configs/test.config

```diff
@@ -0,0 +1,57 @@
+// nf-core label resource requirements
+// check_max function removed for now.
+
+process {
+
+    cpus   = 1
+    memory = 6.GB
+    time   = 4.h
+
+    withLabel: process_low {
+        cpus   = 2
+        memory = 6.GB
+        time   = 4.h
+    }
+    withLabel: process_medium {
+        cpus   = 6
+        memory = 12.GB
+        time   = 24.h
+    }
+    withLabel: process_high {
+        cpus   = 10
+        memory = 32.GB
+        time   = 48.h
+    }
+    withLabel: process_long {
+        time   = 120.h
+    }
+    withLabel: process_high_memory {
+        memory = 32.GB
+    }
+    
+    withName: 'BUILD_MERYL_HIFI_DATABASE:MERYL_COUNT' {
+        cpus   = 3
+        memory = 32.GB
+        time   = 4.h
+    }
+    withName: 'BUILD_MERYL_HIC_DATABASE:MERYL_COUNT' {
+        cpus   = 3
+        memory = 32.GB
+        time   = 4.h
+    }
+    withName: 'BUSCO' {
+        cpus   = 10
+        memory = 12.GB
+        time   = 8.h
+    }
+
+    withName: 'ASSEMBLY_REPORT:QUARTO' {
+    containerOptions = '--platform=linux/amd64'
+    }
+}
+params.fastk.kmer_size         = 14 
+params.meryl.kmer_size         = 14 
+params.genescopefk.kmer_size   = 14
+params.busco.lineages          = 'bacteria_odb10'     //eukaryota_odb10'
+params.organelle_assembly_mode = 'none'
+params.ncbi.taxdb              = '/Users/marpi652/nbis/assembly-project-template-myTest/data/dbs/taxdump.tar.gz'
```

Comment:
This should be defined in another way either using a custom config, or have these settings in the cluster configs.